jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-actions.js

```diff
@@ -22,6 +22,8 @@ define([
 ) {
     'use strict';
 
+    var NS = 'jupyter-dashboard';
+
```

Comment:
This was a remnant from the configurable value that I removed. I thought it still made sense as a constant value, but I will copy it back into the string values.